keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ClusterDeployStrategy.kt

```diff
@@ -6,6 +6,7 @@ import java.time.Duration.ZERO
 sealed class ClusterDeployStrategy {
   open val isStaggered: Boolean = false
   open val stagger: List<StaggeredRegion> = emptyList()
```

Comment:
why is this "Amazon" in a generic API package? Should be "CloudProvider"

Reply:
That whole thing should probably be moved to `keel-ec2-api`. Let me see if I can make that work.